docker-compose.yml

```diff
@@ -48,9 +48,10 @@ services:
     environment:
       - REDIS_INSTANCE_URI=redis://redis-data:6379
       - ACCOUNT_STORE_API_HOST=http://account-store:8080
-      - AUTHENTICATOR_HOST=http://localhost:4004
-      - POST_AWARD_FRONTEND_HOST=http://localhost:4002
-      - POST_AWARD_SUBMIT_HOST=http://localhost:4003
+      - AUTHENTICATOR_HOST=http://authenticator.levellingup.gov.localhost:4004
+      - COOKIE_DOMAIN=.levellingup.gov.localhost
+      - POST_AWARD_FRONTEND_HOST=http://find-monitoring-data.levellingup.gov.localhost:4001
+      - POST_AWARD_SUBMIT_HOST=http://submit-monitoring-data.levellingup.gov.localhost:4001
```

Comment:
Think we can only merge this when the monolith stuff is merged - at the moment none (of the main branches) of the apps legitimately use these domains, I think?

They probably work though ... 🤷

Reply:
Aye.. that's why I targeted `hack/monolith`.

Reply:
Rebased and fixed conflicts.

Reply:
rebased is one word for 'merged that branch into this one' 👀